src/tools/illink/src/ILLink.RoslynAnalyzer/ILLink.RoslynAnalyzer.csproj

```diff
@@ -9,6 +9,8 @@
     <AnalysisLevel>Latest</AnalysisLevel>
     <NoWarn Condition="'$(DotNetBuildFromSource)' == 'true'">$(NoWarn);CS8524</NoWarn>
     <AnalyzerLanguage>cs</AnalyzerLanguage>
+    <!-- This analyzer doesn't follow the convention for generator projects so we set the property manually. -->
+    <IsGeneratorProject>true</IsGeneratorProject>
```

Comment:
Please undo this change. The `IsGeneratorProject` must be set in a Directory.Build.props file (if not inferred automatically based on the project file name): https://github.com/dotnet/runtime/blob/d4db2d96bb0f55211c30dc69ac587b204bb418f3/Directory.Build.props#L445